services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -61,6 +61,11 @@ const UniversalRoutes = (
         <Route path={RoutesRecord.HELP} element={<Help />} />
     </Fragment>
 )
+//Doing this in order to pass the :id for redirecting '/:id' to ':id/edit' for state users viewing rates
+const RateEditRouteRedirect = () => {
+    const { id } = useParams();
+    return <Navigate to={`/${id}/edit`} />
+}
```

Comment:
I don't believe normally a user based redirect would happen at this level. It generally happens at the page level, so in `RateSummary.tsx` . You can use `const navigate = useNavigate()` and `navigate(':/id/edit')` after checking if the user is a state user. There's a somewhat similar example in the codebase [here](https://github.com/Enterprise-CMCS/managed-care-review/blob/main/services/app-web/src/pages/StateSubmission/SubmissionType/SubmissionType.tsx#L208)